{
 "PRed": [
  {
   "PR": {
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1679924224.1623476,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 6,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 7,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 10,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.3"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1334326562.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.3"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2023.04.28.23.49.10",
 "pre_pr_migrator_attempts": {
  "pypy38": 81
 },
 "pre_pr_migrator_status": {
  "pypy38": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/5230795942\">bot CI job</a>): main: ['linux_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - package python-3.8.12-0_73_pypy requires python_abi 3.8 *_pypy38_pp73, but none of the providers can be installed\\n', 'linux_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-main-6.5.0-head6826_1 requires libxcb >=1.13,<1.14.0a0, but none of the providers can be installed\\n', 'linux_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - package python-3.9.10-0_73_pypy requires python_abi 3.9 *_pypy39_pp73, but none of the providers can be installed\\n', 'linux_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-main-6.5.0-head6826_1 requires libxcb >=1.13,<1.14.0a0, but none of the providers can be installed\\n', 'osx_64_python3.10.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'osx_64_python3.11.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'osx_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - package python-3.8.12-0_73_pypy requires python_abi 3.8 *_pypy38_pp73, but none of the providers can be installed\\n', 'osx_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n  - package pyside6-6.4.0-py38hc3b007c_0 requires python_abi 3.8.* *_cp38, but none of the providers can be installed\\n', 'osx_64_python3.8.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'osx_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - package python-3.9.10-0_73_pypy requires python_abi 3.9 *_pypy39_pp73, but none of the providers can be installed\\n', 'osx_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n  - package pyside6-6.4.0-py39h2da4cda_0 requires python_abi 3.9.* *_cp39, but none of the providers can be installed\\n', 'osx_64_python3.9.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h694c41f_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'win_64_python3.10.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'win_64_python3.11.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'win_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - nothing provides vc 9 needed by pyside2-2.0.0.dev0-py27_vc9_0\\n', 'win_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n  - package pyside6-6.4.0-py37h1ccf79b_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'win_64_python3.8.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n', 'win_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - nothing provides vc 9 needed by pyside2-2.0.0.dev0-py27_vc9_0\\n', 'win_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n  - package pyside6-6.4.0-py37h1ccf79b_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'win_64_python3.9.____cpythonqt6: Encountered problems while solving:\\n  - package qt6-advanced-docking-system-4.0.1-h91493d7_0 requires qt6-main >=6.4.2,<6.5.0a0, but none of the providers can be installed\\n']"
 },
 "smithy_version": "3.23.1"
}